records_mover/pandas/__init__.py

```diff
@@ -33,3 +37,14 @@ def purge_unnamed_unused_columns(df: DataFrame) -> DataFrame:
             if not df[column].notnull().any():
                 df = df.drop(column, axis=1)
     return df
+
+
+def convert_integer_dtypes(df: 'DataFrame'):
+    # TODO: Document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker